AGENTS.md

```
@@ -286,10 +324,30 @@ When changing any GitHub Actions workflows (.github/workflows/*.yml) or actions
      run: gh command "$MY_VAR"
    ```
 
-5. **Re-run actionlint** after making changes to ensure all issues are resolved.
+ 5. **Re-run actionlint** after making changes to ensure all issues are resolved.
```

markdownlint-cli2 (0.23.2)

[warning] Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)